winbuild/config.py

```diff
@@ -15,75 +15,63 @@
 libs = {
     # 'openjpeg': {
     #     'filename': 'openjpeg-2.0.0-win32-x86.zip',
-    #     'hash': 'sha1:xxx',
     #     'version': '2.0'
     # },
     'zlib': {
         'url': 'http://zlib.net/zlib1211.zip',
         'filename': PILLOW_DEPENDS_DIR + 'zlib1211.zip',
-        'hash': 'md5:16b41357b2cd81bca5e1947238e64465',
         'dir': 'zlib-1.2.11',
     },
     'jpeg': {
         'url': 'http://www.ijg.org/files/jpegsr9b.zip',
         'filename': PILLOW_DEPENDS_DIR + 'jpegsr9b.zip',
-        'hash': 'md5:a21b8024d78ba05857a75272b4fa95ec',  # not found - generated by wiredfool
         'dir': 'jpeg-9b',
     },
     'tiff': {
         'url': 'ftp://ftp.remotesensing.org/pub/libtiff/tiff-4.0.7.zip',
```

Comment:
This reference URL is 404 (see https://en.wikipedia.org/wiki/Libtiff#Website_hijacking), the new site is ftp://download.osgeo.org/libtiff